src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+function conversion(str) {
+  return str.dataset.salary.split(',').join('').slice(1, -1);
+}
+
+function sotrSalary() {
+  const arrayEmployees = [...document.querySelectorAll('li')]
```

Comment:
```suggestion
  const arrayEmployees = [...document.querySelectorAll('li')]
```
Do not use name of data structures or data types in the names of yours variables